I, [2018-01-09T14:17:37.423862 #30189]  INFO -- test-ubuntu-1604: -----> Converging <test-ubuntu-1604>...
I, [2018-01-09T14:17:37.424709 #30189]  INFO -- test-ubuntu-1604: Preparing files for transfer
I, [2018-01-09T14:17:37.424841 #30189]  INFO -- test-ubuntu-1604: Preparing playbook
I, [2018-01-09T14:17:37.425098 #30189]  INFO -- test-ubuntu-1604: Preparing inventory
I, [2018-01-09T14:17:37.425192 #30189]  INFO -- test-ubuntu-1604: Preparing modules
I, [2018-01-09T14:17:37.425338 #30189]  INFO -- test-ubuntu-1604: nothing to do for modules
I, [2018-01-09T14:17:37.425394 #30189]  INFO -- test-ubuntu-1604: Preparing roles
I, [2018-01-09T14:17:37.429518 #30189]  INFO -- test-ubuntu-1604: Preparing ansible.cfg file
I, [2018-01-09T14:17:37.429642 #30189]  INFO -- test-ubuntu-1604: Empty ansible.cfg generated
I, [2018-01-09T14:17:37.429735 #30189]  INFO -- test-ubuntu-1604: Preparing group_vars
I, [2018-01-09T14:17:37.429857 #30189]  INFO -- test-ubuntu-1604: nothing to do for group_vars
I, [2018-01-09T14:17:37.429906 #30189]  INFO -- test-ubuntu-1604: Preparing additional_copy_path
I, [2018-01-09T14:17:37.430013 #30189]  INFO -- test-ubuntu-1604: Preparing host_vars
I, [2018-01-09T14:17:37.430135 #30189]  INFO -- test-ubuntu-1604: nothing to do for host_vars
I, [2018-01-09T14:17:37.430198 #30189]  INFO -- test-ubuntu-1604: Preparing hosts file
I, [2018-01-09T14:17:37.430311 #30189]  INFO -- test-ubuntu-1604: Preparing spec
I, [2018-01-09T14:17:37.430413 #30189]  INFO -- test-ubuntu-1604: nothing to do for spec
I, [2018-01-09T14:17:37.430461 #30189]  INFO -- test-ubuntu-1604: Preparing library plugins
I, [2018-01-09T14:17:37.430594 #30189]  INFO -- test-ubuntu-1604: nothing to do for library plugins
I, [2018-01-09T14:17:37.430643 #30189]  INFO -- test-ubuntu-1604: Preparing callback plugins
I, [2018-01-09T14:17:37.431593 #30189]  INFO -- test-ubuntu-1604: nothing to do for callback plugins
I, [2018-01-09T14:17:37.431673 #30189]  INFO -- test-ubuntu-1604: Preparing filter_plugins
I, [2018-01-09T14:17:37.431784 #30189]  INFO -- test-ubuntu-1604: nothing to do for filter_plugins
I, [2018-01-09T14:17:37.431820 #30189]  INFO -- test-ubuntu-1604: Preparing lookup_plugins
I, [2018-01-09T14:17:37.431906 #30189]  INFO -- test-ubuntu-1604: nothing to do for lookup_plugins
I, [2018-01-09T14:17:37.431955 #30189]  INFO -- test-ubuntu-1604: Preparing ansible vault password
I, [2018-01-09T14:17:37.432010 #30189]  INFO -- test-ubuntu-1604: Preparing additional_ssh_private_keys
I, [2018-01-09T14:17:37.432060 #30189]  INFO -- test-ubuntu-1604: nothing to do for additional_ssh_private_keys
I, [2018-01-09T14:17:37.432086 #30189]  INFO -- test-ubuntu-1604: Finished Preparing files for transfer
I, [2018-01-09T14:17:37.432237 #30189]  INFO -- test-ubuntu-1604: Installing ansible, will try to determine platform os
I, [2018-01-09T14:17:47.441186 #30189]  INFO -- test-ubuntu-1604: [SSH] connection failed, retrying in 1 seconds (#<Net::SSH::ConnectionTimeout: Net::SSH::ConnectionTimeout>)
W, [2018-01-09T14:17:58.452554 #30189]  WARN -- test-ubuntu-1604: [SSH] connection failed, terminating (#<Net::SSH::ConnectionTimeout: Net::SSH::ConnectionTimeout>)
E, [2018-01-09T14:17:58.456072 #30189] ERROR -- test-ubuntu-1604: Converge failed on instance <test-ubuntu-1604>.
E, [2018-01-09T14:17:58.456233 #30189] ERROR -- test-ubuntu-1604: ------Exception-------
E, [2018-01-09T14:17:58.456272 #30189] ERROR -- test-ubuntu-1604: Class: Kitchen::ActionFailed
E, [2018-01-09T14:17:58.456297 #30189] ERROR -- test-ubuntu-1604: Message: SSH session could not be established
E, [2018-01-09T14:17:58.456316 #30189] ERROR -- test-ubuntu-1604: ----------------------
E, [2018-01-09T14:17:58.456331 #30189] ERROR -- test-ubuntu-1604: ------Backtrace-------
E, [2018-01-09T14:17:58.456345 #30189] ERROR -- test-ubuntu-1604: /home/john/.rvm/gems/ruby-2.4.0/gems/test-kitchen-1.19.2/lib/kitchen/provisioner/base.rb:83:in `rescue in call'
E, [2018-01-09T14:17:58.456360 #30189] ERROR -- test-ubuntu-1604: /home/john/.rvm/gems/ruby-2.4.0/gems/test-kitchen-1.19.2/lib/kitchen/provisioner/base.rb:86:in `call'
E, [2018-01-09T14:17:58.456381 #30189] ERROR -- test-ubuntu-1604: /home/john/.rvm/gems/ruby-2.4.0/gems/test-kitchen-1.19.2/lib/kitchen/instance.rb:393:in `block in converge_action'
E, [2018-01-09T14:17:58.456395 #30189] ERROR -- test-ubuntu-1604: /home/john/.rvm/gems/ruby-2.4.0/gems/test-kitchen-1.19.2/lib/kitchen/instance.rb:536:in `synchronize_or_call'
E, [2018-01-09T14:17:58.456407 #30189] ERROR -- test-ubuntu-1604: /home/john/.rvm/gems/ruby-2.4.0/gems/test-kitchen-1.19.2/lib/kitchen/instance.rb:498:in `block in action'
E, [2018-01-09T14:17:58.456420 #30189] ERROR -- test-ubuntu-1604: /usr/share/rvm/rubies/ruby-2.4.0/lib/ruby/2.4.0/benchmark.rb:293:in `measure'
E, [2018-01-09T14:17:58.456432 #30189] ERROR -- test-ubuntu-1604: /home/john/.rvm/gems/ruby-2.4.0/gems/test-kitchen-1.19.2/lib/kitchen/instance.rb:497:in `action'
E, [2018-01-09T14:17:58.456443 #30189] ERROR -- test-ubuntu-1604: /home/john/.rvm/gems/ruby-2.4.0/gems/test-kitchen-1.19.2/lib/kitchen/instance.rb:389:in `converge_action'
E, [2018-01-09T14:17:58.456456 #30189] ERROR -- test-ubuntu-1604: /home/john/.rvm/gems/ruby-2.4.0/gems/test-kitchen-1.19.2/lib/kitchen/instance.rb:368:in `block in transition_to'
E, [2018-01-09T14:17:58.456468 #30189] ERROR -- test-ubuntu-1604: /home/john/.rvm/gems/ruby-2.4.0/gems/test-kitchen-1.19.2/lib/kitchen/instance.rb:367:in `each'
E, [2018-01-09T14:17:58.456479 #30189] ERROR -- test-ubuntu-1604: /home/john/.rvm/gems/ruby-2.4.0/gems/test-kitchen-1.19.2/lib/kitchen/instance.rb:367:in `transition_to'
E, [2018-01-09T14:17:58.456490 #30189] ERROR -- test-ubuntu-1604: /home/john/.rvm/gems/ruby-2.4.0/gems/test-kitchen-1.19.2/lib/kitchen/instance.rb:135:in `converge'
E, [2018-01-09T14:17:58.456501 #30189] ERROR -- test-ubuntu-1604: /home/john/.rvm/gems/ruby-2.4.0/gems/test-kitchen-1.19.2/lib/kitchen/command.rb:197:in `public_send'
E, [2018-01-09T14:17:58.456512 #30189] ERROR -- test-ubuntu-1604: /home/john/.rvm/gems/ruby-2.4.0/gems/test-kitchen-1.19.2/lib/kitchen/command.rb:197:in `run_action_in_thread'
E, [2018-01-09T14:17:58.456523 #30189] ERROR -- test-ubuntu-1604: /home/john/.rvm/gems/ruby-2.4.0/gems/test-kitchen-1.19.2/lib/kitchen/command.rb:169:in `block (2 levels) in run_action'
E, [2018-01-09T14:17:58.456534 #30189] ERROR -- test-ubuntu-1604: ----End Backtrace-----
E, [2018-01-09T14:17:58.456545 #30189] ERROR -- test-ubuntu-1604: ---Nested Exception---
E, [2018-01-09T14:17:58.456555 #30189] ERROR -- test-ubuntu-1604: Class: Kitchen::Transport::SshFailed
E, [2018-01-09T14:17:58.456566 #30189] ERROR -- test-ubuntu-1604: Message: SSH session could not be established
E, [2018-01-09T14:17:58.456576 #30189] ERROR -- test-ubuntu-1604: ----------------------
E, [2018-01-09T14:17:58.456586 #30189] ERROR -- test-ubuntu-1604: ------Backtrace-------
E, [2018-01-09T14:17:58.456597 #30189] ERROR -- test-ubuntu-1604: /home/john/.rvm/gems/ruby-2.4.0/gems/test-kitchen-1.19.2/lib/kitchen/provisioner/base.rb:83:in `rescue in call'
E, [2018-01-09T14:17:58.456607 #30189] ERROR -- test-ubuntu-1604: /home/john/.rvm/gems/ruby-2.4.0/gems/test-kitchen-1.19.2/lib/kitchen/provisioner/base.rb:86:in `call'
E, [2018-01-09T14:17:58.456618 #30189] ERROR -- test-ubuntu-1604: /home/john/.rvm/gems/ruby-2.4.0/gems/test-kitchen-1.19.2/lib/kitchen/instance.rb:393:in `block in converge_action'
E, [2018-01-09T14:17:58.456629 #30189] ERROR -- test-ubuntu-1604: /home/john/.rvm/gems/ruby-2.4.0/gems/test-kitchen-1.19.2/lib/kitchen/instance.rb:536:in `synchronize_or_call'
E, [2018-01-09T14:17:58.456640 #30189] ERROR -- test-ubuntu-1604: /home/john/.rvm/gems/ruby-2.4.0/gems/test-kitchen-1.19.2/lib/kitchen/instance.rb:498:in `block in action'
E, [2018-01-09T14:17:58.456651 #30189] ERROR -- test-ubuntu-1604: /usr/share/rvm/rubies/ruby-2.4.0/lib/ruby/2.4.0/benchmark.rb:293:in `measure'
E, [2018-01-09T14:17:58.456661 #30189] ERROR -- test-ubuntu-1604: /home/john/.rvm/gems/ruby-2.4.0/gems/test-kitchen-1.19.2/lib/kitchen/instance.rb:497:in `action'
E, [2018-01-09T14:17:58.456674 #30189] ERROR -- test-ubuntu-1604: /home/john/.rvm/gems/ruby-2.4.0/gems/test-kitchen-1.19.2/lib/kitchen/instance.rb:389:in `converge_action'
E, [2018-01-09T14:17:58.456689 #30189] ERROR -- test-ubuntu-1604: /home/john/.rvm/gems/ruby-2.4.0/gems/test-kitchen-1.19.2/lib/kitchen/instance.rb:368:in `block in transition_to'
E, [2018-01-09T14:17:58.456701 #30189] ERROR -- test-ubuntu-1604: /home/john/.rvm/gems/ruby-2.4.0/gems/test-kitchen-1.19.2/lib/kitchen/instance.rb:367:in `each'
E, [2018-01-09T14:17:58.456711 #30189] ERROR -- test-ubuntu-1604: /home/john/.rvm/gems/ruby-2.4.0/gems/test-kitchen-1.19.2/lib/kitchen/instance.rb:367:in `transition_to'
E, [2018-01-09T14:17:58.456722 #30189] ERROR -- test-ubuntu-1604: /home/john/.rvm/gems/ruby-2.4.0/gems/test-kitchen-1.19.2/lib/kitchen/instance.rb:135:in `converge'
E, [2018-01-09T14:17:58.456735 #30189] ERROR -- test-ubuntu-1604: /home/john/.rvm/gems/ruby-2.4.0/gems/test-kitchen-1.19.2/lib/kitchen/command.rb:197:in `public_send'
E, [2018-01-09T14:17:58.456752 #30189] ERROR -- test-ubuntu-1604: /home/john/.rvm/gems/ruby-2.4.0/gems/test-kitchen-1.19.2/lib/kitchen/command.rb:197:in `run_action_in_thread'
E, [2018-01-09T14:17:58.456769 #30189] ERROR -- test-ubuntu-1604: /home/john/.rvm/gems/ruby-2.4.0/gems/test-kitchen-1.19.2/lib/kitchen/command.rb:169:in `block (2 levels) in run_action'
E, [2018-01-09T14:17:58.456788 #30189] ERROR -- test-ubuntu-1604: ----End Backtrace-----
